### ISSUE TITLE
Add missing proto files

### ISSUE THREAD

**Issue by benthecarman** [Kixunil/tonic_lnd#15]
This only includes the lightning.proto file. Would be beneficial to have all the missing protos to have access to the full lnd api

**Comment by Kixunil** [comment]
Yeah, someone talked to me about this over e-mail. Doesn't seem to be trivial but probably doable.

**Comment by yzernik** [comment]
@benthecarman I made a fork of this repo with support for the full LND API, and published the crate here in case anyone wants to use it: https://crates.io/crates/tonic_openssl_lnd

**Comment by Kixunil** [comment]
FYI, I want to review your PR, but it's quite large and I don't have much time and mental energy to focus on such a large thing. Perhaps splitting it in two parts - ssl and more proto files would help me but I can't promise it will.

**Comment by yzernik** [comment]
@Kixunil Sure, I'll try to split it into a few small PR's. My intention was not to create a competing library, but I just needed these features urgently. I'll probably have some time to do this next week.
